tests/unit/FilingHistoryList1.spec.ts

```diff
@@ -378,6 +378,11 @@ describe('Filing History List - misc functionality', () => {
     jest.spyOn(vm, 'isAllowed').mockReturnValue(true)
     expect(vm.disableCorrection({ ...item })).toBe(false)
 
+    // conditions[3]: FE filing that is Completed or Corrected
+    for (const status of ['COMPLETED', 'CORRECTED']) {
+      expect(vm.disableCorrection({ ...item, isFutureEffective: true, status })).toBe(false)
+    }
```

Comment:
Allow correcting a Completed or Corrected FE filing.

Reply:
If any other (uncommon) state needs to be corrected, it could be done as a one-of.